docs/builds/guides/integration/advanced-setup.md

```diff
@@ -148,22 +149,30 @@ module.exports = {
 							}
 						}
 					},
+					'css-loader',
 					{
 						loader: 'postcss-loader',
-						options: styles.getPostCssConfig( {
-							themeImporter: {
-								themePath: require.resolve( '@ckeditor/ckeditor5-theme-lark' )
-							},
-							minify: true
-						} )
-					},
+						options: { 
+						    postcssOptions: styles.getPostCssConfig( {
+                                themeImporter: {
+                                    themePath: require.resolve( '@ckeditor/ckeditor5-theme-lark' )
+                                },
+                                minify: true
+                            } )
+                        }
+					}
 				]
 			}
 		]
 	}
 };
 ```
 
+<info-box>
```

Comment:
We need to work on the wording here.

But first, I'd like to ensure that:

*   If I change `webpack@5` to `webpack@4` in the `npm i` step **and nothing else**, everything will work fine when I follow this guide? That's how I understand this comment and that'd be great, however...
*   Do we then need the branching for `postcss-loader@4` vs `postcss-loader@3` change in the CRA guide?